src/rastervision/tagging/tasks/predict.py

```diff
@@ -87,7 +88,27 @@ def compute_probs(run_path, model, options, generator, split):
         normalize=True, only_xy=False)
 
     for batch_ind, batch in enumerate(split_gen):
-        y_probs.append(model.predict(batch.x))
+        # Performing safe augmentations on images one by one, predicting
+        # probs and taking average, if calculating test probabilities
+        if split == TEST:
+            for img_ind in range(batch.x.shape[0]):
+                img = batch.x[img_ind, :, :, :]
+                rot_batch = []
```

Comment:
Instead of `rot_batch` and `rot_probs`, `aug_batch` and `aug_probs` would be more accurate names.